_config.yml

```diff
@@ -174,6 +174,22 @@ archive_pagination: true
 category_pagination: true
 tag_pagination: true
 
+# Gitment - A github issue comment tool.
+# Repository: https://github.com/imsun/gitment
+# Demo: https://imsun.github.io/gitment/
+# Introduction: https://imsun.net/posts/gitment-introduction/
+# Quick link to setup a new Github application: https://github.com/settings/applications/new
+# gitment:
+#   # Switch
+#   enable: false
+#   # Your Github ID here (Github username):
+#   githubID:
```

Comment:
use snake case in yaml file

Reply:
Sorry about that, I didn't even notice about that. 👍 
My habit prefers CamelCase.